### PR TITLE
Update color scheme

### PR DESCRIPTION
**Pull request by AliceR**
Centralize data viz colors in theme, making it easier to match colors on map and legend.

Also, changes the color of the bank layer and fixes the label.

### CODE REVIEW

**Review by geohacker — CHANGES_REQUESTED** [review]
image

I think Bank, MFI and Sacco are still kind of close and hard to distinguish. Can we use some potentially very diverging colours? For example https://colorbrewer2.org/#type=diverging&scheme=RdGy&n=4

**Comment by AliceR** [comment]
@geohacker I kept the financial services all with a shade of green, in order to be able to use other hues for the other categories (blues for health, pinkish for education, and the highlight orange for energy. Using a sequential color scheme within one of the categories takes too much color space away from the others. 

Also worth noting that using luminance within a hue creates a visual ranking. Might be desirable for health types (type 4 in lighter shade, less prominent?). Not sure if such ranking applies to the financial services - I tried to avoid it so far.

**Comment by AliceR** [comment]
This is the current color palette:  https://coolors.co/304ca2-f98e08-6a9fb4-54a06f-679289-c8d6af-db93b0

**Comment by geohacker** [comment]
@AliceR thank you. I took another stab at really diverging the colour scheme. They may not be great to look altogether but it makes it easy to distinguish which I think is that they want. 

image
_everything_

image
_just financial_

**Comment by AliceR** [comment]
@geohacker I'll be honest, I don't like the random "confetti" look. But indeed, it does allow for the greatest differentiation between any given layer - Which seems to be considered more important than associative grouping.

Also, changing the colors should be easy enough to revisit by a maintainer later.
Can't approve my PR, so I'll leave it here: ✅

**Comment by geohacker** [comment]
@AliceR definitely agree with you that the confetti look isn't aesthetically pleasing at all, but for now I'm inclined to just go with what allows for greater differentiation. Thank you!

File: src/components/Map.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function Map({ center, zoom, children }) {
   useLayoutEffect(() => {
     const m = new mapbox.Map({
       container: ref.current,
+      // style: 'mapbox://styles/iandmuir/ckhom8rti07cy19pclrmewimq', // use this for high contrast roads
```
Comment (AliceR) [inline src/components/Map.js:17]: 👍

**Pull request merged**